.gitignore

```diff
@@ -7,6 +7,7 @@
 .cache
 *.orig
 *.log
+*.un~
```

Comment:
What is this?

Reply:
It's a vim undo tree file. (Commenting late for hubgit testing)

Reply:
...Extra reply for testing

Reply:
Checking if unresolved stuff works.